ee/localserver/server.go

```diff
@@ -214,36 +213,26 @@ func (ls *localServer) runAsyncdWorkers() time.Time {
 }
 
 func (ls *localServer) Start() error {
-	// Spawn background workers. This loop is a bit weird on startup. We want to populate this data as soon as we can, but because the underlying launcher
-	// run group isn't ordered, this is likely to happen before querier is ready. So we retry at a frequent interval for a couple of minutes, then we drop
-	// back to a slower poll interval. Note that this polling is merely a check against time, we don't repopulate this data nearly so often. (But we poll
+	// Spawn background workers. This loop is a bit weird on startup. We want to populate this data as soon as we can,
```

Comment:
Comment is out of date -- we're not doing this as soon as we can

Reply:
I'll update this a bit more, thanks!